test/Streamly/System/Process.hs

```diff
@@ -123,9 +126,9 @@ createExecutable file content = do
 
 createExecutables :: IO ()
 createExecutables = do
-    createExecutable executableFile executableFileContent
-    createExecutable executableFileFail executableFileFailContent
-    createExecutable executableFilePass executableFilePassContent
+    createExecutable executableFile (prependShebang executableFileContent)
+    createExecutable executableFileFail (prependShebang executableFileFailContent)
+    createExecutable executableFilePass (prependShebang executableFilePassContent)
```

Comment:
We have made these tests unnecessarily complex. Let's just create these files statically in the repository and commit them instead of creating and removing every time we test.

Reply:
There is a way to handle files related to tests and benchmarks properly in the cabal workflow. I don't seem to recall what it is.